src/pages/experimental/uploading/index.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Uploading
+---
+
+### Transferring a resource from a local system to a remote system.
+
+<ImageComponent cols="8">
```

Comment:
```suggestion
<Row>
<Column colLg={8}>
```

The `<ImageComponent>` was deprecated with the release of our Gatsby theme. You can now just place the image in an 8 column wrapper. This docs site has more detail about the new components: https://gatsby-theme-carbon.now.sh/components/markdown#images